draft-ietf-shmoo-remote-fee.md

```diff
@@ -123,13 +123,20 @@ and other roles in the IETF.
 
 These and other operating costs of the IETF are also cross-financed by income generated through
 meeting fees. The intention of this document and the principle stated herein is not to make participation
-free for everyone but to always have a free option that can be used without any barriers other than
-the registration procedure itself. As long as there are still enough paying participants to cover
-the base costs, additional participant can effectively be added without increasing expenses.
+free for everyone but to always have a free option that a potential
+attendee can apply for without any barriers other than
+the registration procedure itself.
+Note that as long as the meeting costs are otherwise funded,
+additional remote participants can usually be added without increasing expenses.
 
 It is not in scope for this document or the SHMOO working group to make suggestions for changing
 the IETF's overall funding model. This is the responsibility of the IETF LLC Board taking agreed principles
 like the one proposed in this document into account.
+Of course, part of this responsibility can include putting a limitation on the amount
+of free participation.
+One possible reason for doing so is the concern that unlimited free remote
+participation might adversely affect the number of paying participants.
```

Comment:
```suggestion
If free remote participation is determined to adversely affect the number of paying participants
or the cost of free participation emerges to a signification factor,
the LLC might implement additional measures to limit on the amount of free registrations.
However, as discussed in the next section, assessment of eligibility is hard and a general limit
in the number of available free registrations can impact
eligible people unfairly and therefore impact openness negatively.
```

This is my proposal. I think this give the LLC some room to do something if needed, however, it also discusses a bit the negative impact of a limit, which I find important.

Reply:
As proposed by Lars we could add another sentence saying something like:

`LLC is expected to publicly announce if and why any measures to limit free registrations are implemented.`

Reply:
I took your change and added a version of Lars's text as an additional commit.